source/language/directives.rst

```diff
@@ -8,3 +8,103 @@ that give additional information to compiler passes and the target
 system or simulator. Ideally the meaning of the program does not change
 if some or all of the directives are ignored, so they can be interpreted
 at the discretion of the consuming process.
+<<<<<<< Updated upstream
```

Comment:
This looks like a lingering merge conflict?

Reply:
Okay, fixed.